### PR TITLE
Add RTL

### PR DESCRIPTION
**Pull request by nixbitcoin**
Doesn't support multi-node setups to reduce the complexity of the initial PR.

### CODE REVIEW

File: pkgs/cl-rest/result [hunk 1]
```diff
@@ -0,0 +1 @@
+/nix/store/x0qckyyr6jdxb1wvqjhkwqgk5zrmbgpz-node_c-lightning-rest-0.5.2
```
Comment (mmilata) [inline pkgs/cl-rest/result:1]: Leftover symlink?

**Review by mmilata — APPROVED** [review]
Works great! Tested: LND, Onion service; not tested: Loop, network namespaces.

**Comment by nixbitcoin** [comment]
Can I squash the fixups like this @erikarvstedt?

**Comment by nixbitcoin** [comment]
I made the `onionService` enabled by default like in `spark-wallet`. Anything else would overcomplicate it for the user IMO.

**Comment by erikarvstedt** [comment]
Yes, please squash the fixups.

**Comment by nixbitcoin** [comment]
> Yes, please squash the fixups.

Done

**Comment by erikarvstedt** [comment]
Excellent work. [Fixups™](https://github.com/erikarvstedt/nix-bitcoin/commits/rtl-ea).

**Review by erikarvstedt — APPROVED** [review]
ACK 030f649009b35ce3c59ce25bbdb48238830a234c

**Comment by jonasnick** [comment]
Cool, thanks everyone.

**Pull request merged**